rebar.config

```diff
@@ -1,14 +1,16 @@
 {erl_opts,
  [warn_unused_import, warn_export_vars, warnings_as_errors, verbose, report, debug_info]}.
 
-{minimum_otp_vsn, "23"}.
+{minimum_otp_vsn, "24"}.
 
 {cover_enabled, true}.
 
 {cover_opts, [verbose]}.
 
 {deps, []}.
 
+{overrides, [{override, [{erl_opts, []}]}]}.
```

Comment:
I mean, maybe (?) For the dependencies I didn't test how e.g. not having "warnings as errors" this would work, but I can make them the same, and check the results. Thanks, @elbrujohalcon. Will do that.